src/components/DisplayCurrentPlaylist.js

```diff
@@ -0,0 +1,40 @@
+/*
+
+  This Component displays the current playlists.
+
+  props - current playlist , allSongs
+*/
+
+import { playlistType, songType } from "../lib/types.js";
+import PropTypes from "prop-types";
+import styles from "../styles/DisplayCurrentPlaylist.module.css";
+
+export default function DisplayCurrentPlaylist({ playlist, allSongs }){
+
+const currentSongs = allSongs.filter((song) => song.playlistID === playlist.id);
+
+currentSongs.map((song) => song.timeAdded = new Date(song.timeAdded));
```

Comment:
What is the purpose of this line? Remember that `map` returns a list, so you may want to use `forEach` here. But I'm not really sure what this code is trying to do.

Reply:
We were dealing with the standard `.format()` and converting it to 12 hour time. I believe `map` was necessary to use `.toLocaleString()`. Changed the `SongInput` to take the time using `.format("LT")` which gives us the time display we desired so I removed the `map`